### PR TITLE
Update classes.md

### PR DESCRIPTION
**Pull request by DipanshKhandelwal**

### CODE REVIEW

File: pages/docs/reference/classes.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ Creating instances of nested, inner and anonymous inner classes is described in
 
 Classes can contain
 
-* Constructors and initializer blocks
```
Comment (yole) [inline pages/docs/reference/classes.md]: Please format the link correctly.

**Comment by DipanshKhandelwal** [comment]
The link is correct, the part after # is the internal link for the page.

**Comment by yole** [comment]
The reference is correct, but this is not a link. See the [Markdown syntax documentation](https://daringfireball.net/projects/markdown/syntax#link) for the right way to format links.

**Comment by DipanshKhandelwal** [comment]
Oh sorry, I got it now.

**Pull request merged**